src/symbolize/mod.rs

```diff
@@ -78,7 +84,7 @@ impl Symbol {
     /// unix platforms other than OSX) and when a binary is compiled with
     /// debuginfo. If neither of these conditions is met then this will likely
     /// return `None`.
-    pub fn filename(&self) -> Option<&Path> {
+    pub fn filename(&self) -> Option<BytesOrWideString> {
```

Comment:
I'd personally prefer to not have a breaking change to this library right now, so could this method be left as-is but a new API added along the lines of `filename_raw`?